DevCycle.SDK.Server.Local.Example/Program.cs

```diff
@@ -70,7 +71,7 @@ private static async Task ClientInitialized(DevCycleUser user)
                 Console.WriteLine(entry.Key + " : " + entry.Value);
             }
 
-            Console.WriteLine((await api.Variable(user, "test-variable", true)).ToString());
```

Comment:
cleanup

Reply:
can keep it here, its in the example app